<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Visualization</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="80,20">
   <item>
    <layout class="QVBoxLayout" name="plot_layout"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0,50,0">
     <item>
      <widget class="QListWidget" name="chart_types">
       <property name="currentRow">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Line Plot</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Scatter Plot</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Stack Plot</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bar Chart</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pie Chart</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Histogram</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Heat Map</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Box Plot</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Event Plot</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Network Graph</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labely">
       <property name="text">
        <string>Y-Axis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="dropdown_yaxis"/>
     </item>
     <item>
      <widget class="QLabel" name="labelx">
       <property name="text">
        <string>X-Axis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="dropdown_xaxis"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_plot">
       <property name="text">
        <string>Generate Plot</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelgroupby">
       <property name="text">
        <string>Group by:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="dropdown_groupby"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_multipleplots">
       <property name="text">
        <string>Show Grouped Plots</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelstatus">
       <property name="text">
        <string>Ready.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
